pandas/_libs/groupby.pyx

```diff
@@ -869,7 +869,9 @@ def group_last(rank_t[:, :] out,
 
     assert min_count == -1, "'min_count' only used in add and prod"
 
-    if not len(values) == len(labels):
+    # NOTE:
+    # Casting to avoid build warnings
+    if not len(values) == <Py_ssize_t>len(labels):
```

Comment:
Is it possible to get the same result without casting? Maybe labels.shape[0] for memory views?

Reply:
@WillAyd It works!

---

Just so I know for future times, are we trying to avoid casting?

or casting is not possible with memory views?

Reply:
If there's an easily available approach that doesn't require casting then should almost always go that route